templates/partial/snippet.adoc

```diff
@@ -0,0 +1,3 @@
+// A text snippet is a section of text that is stored in an AsciiDoc file.
```

Comment:
`snip` or `snippet`
BTW, we've been using `snip`.

Reply:
???

Reply:
I meant the `snip_...adoc` file-name convention in our docs:
https://github.com/eclipse-che/che-docs/tree/main/modules/administration-guide/examples

Anyway, I guess it shouldn't matter in this file.